common4j/src/main/com/microsoft/identity/common/java/commands/parameters/BrokerSilentTokenCommandParameters.java

```diff
@@ -56,12 +55,6 @@ public void validate() throws ArgumentException {
                     "mCallerUId", "Caller Uid is not set"
             );
         }
-        if (StringUtil.isNullOrEmpty(callerPackageName)) {
```

Comment:
why removing this check?

Reply:
I'm actually not sure at this point.  I'll go back and check if there was a reason.  It may have been a whoops.

Reply:
I moved these all the way down to CommandParameters.... so if we want a validation check on these It should be there.  I'll see if it makes sense to add it there.